features/step_definitions/edit_dob_ssn_steps.rb

```diff
@@ -25,14 +25,27 @@
   end
 end
 
-When("user enters an invalid SSN and clicks on update") do
-  find('#person_ssn').set("212-31-31")
+When("Admin enters a invalid SSN and clicks on update") do
+  fill_in 'person[ssn]', :with => ''
+  fill_in 'person[ssn]', :with => '212-31-31'
   sleep 1
   page.find_button("Update").click
   page.driver.browser.switch_to.alert.accept
   sleep 1
 end
 
+When(/^Admin will see the Edit DOB SSN button$/) do
+  find_link('Edit DOB / SSN')
+end
+
+When(/^Admin clicks Action for a person on families index page$/) do
+  find('.dropdown.pull-right', text: 'Actions').click
+end
+
+When(%r{^Admin clicks on edit DOB/SSN link$}) do
+  click_link('Edit DOB / SSN')
```

Comment:
hard-coded english needs to be removed

Reply:
will make changes to remove hard coded english